fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/virtualcolumns/ExpressionVirtualColumn.java

```diff
@@ -70,4 +72,17 @@ public ExpressionVirtualColumn withExpression(String expression) {
     public ExpressionVirtualColumn withOutputType(String outputType) {
         return new ExpressionVirtualColumn(getName(), getExpression(), outputType);
     }
+
+    @Override
+    public boolean virtualColumnEquals(Object o) {
+        if (this == o) { return true; }
+        if (!(o instanceof ExpressionVirtualColumn)) { return false; }
```

Comment:
If this is equals it will trigger automatic equals testing, so no need to test this.